### PR TITLE
19 screens

### PR DESCRIPTION
**Pull request by kosara-g**
User story 19 implemented. New features:
* Welcome screen on startup (scales with application).
* Only import button is enabled on startup.
* Game over screen when the simulation is finished (scales with application).
* Game over screen works with stepping back and forth in the simulation.

### CODE REVIEW

**Review by ghost — APPROVED** [review]
stuff is legit af

File: Application/src/model/Menu.java [hunk 1]
```diff
@@ -217,6 +221,7 @@ private void setEvents(Stage stage) {
 
         button_previous.setOnAction((ActionEvent event) ->{
             Net.push("{" + DiagramView.getDiagramViewInView().getTab().getId() + ", previous_message}");
+            Draw.removeGameOver();  // Removes the game over screen from the canvas.
```
Comment (pontuslaestadius) [inline Application/src/model/Menu.java]: This assumes that you can go back. Should this action be in the decode?

File: Application/src/view/visuals/Draw.java [hunk 2]
```diff
@@ -257,6 +278,21 @@ private void renderClass() {
...
     }
 
+    /**
+     * Updates the game over message to fit the resized window.
+     */
+    private void renderGameOver() {
+        if (gameOverNotification == null){
+            return;  // The game over message hasn't been initialized yet.
+        }
+        else {
```
Comment (pontuslaestadius) [inline Application/src/view/visuals/Draw.java]: Else not required since if statement breaks flow.

File: Application/src/view/visuals/Draw.java [hunk 3]
```diff
@@ -13,6 +13,7 @@
 import java.util.ArrayList;
 
 /**
+ * @version 1.5
```
Comment (IsabelleTornqvist) [inline Application/src/view/visuals/Draw.java]: two version tags

**Review by timjon — CHANGES_REQUESTED** [review]
Does not work with tabs

**Comment by kosara-g** [comment]
Closed due to lack of time. Will open a new pull request with working feature.

**Pull request closed without merging**